game/static/game/css/landing.css

```
@@ -730,6 +730,82 @@ body {
     border-color: rgba(240, 192, 64, 0.3);
     transform: translateY(-3px);
 }
+/*BACK TO TOP BUTTON*/
```

Stylelint (17.12.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)